### ISSUE TITLE
Sometimes the MUI Text Field overwrites content that was programatically entered with the component "label"

### ISSUE THREAD

**Issue by SailingSteve** [wevote/WebApp#3507]
Part of this task is to figure out how to reproduce the problem, the second is to fix it.  I think this is a known issue for MUI.
```
    <StyledTextField
        type="text"
        label="Name Shown with Endorsements"
        margin="dense"
        autoComplete="organization"
        variant="outlined"
        id={`organization-name-${externalUniqueId}`}
        name="organizationName"
        placeholder="How would you like your name displayed publicly?"
        onKeyDown={this.handleKeyPressOrganizationName}
        onChange={this.updateOrganizationName}
        value={organizationName}
    />
```

**Comment by SailingSteve** [comment]
<img width="375" alt="Screen Shot 2022-08-24 at 2 38 12 PM" src="https://user-images.githubusercontent.com/6923653/186528329-9128bb52-1dca-4620-aebb-fda5c233a5bf.png">

**Comment by DaleMcGrew** [comment]
Thank you @fdiazq for researching this, and providing the following notes. I am closing this at your recommendation.
=======
The issue involves text being overwritten from StyledTextField.
I tested the profile settings page to attempt to reproduce the issue , and  I haven't been able to reproduce it.
I tested on Chrome,Safari, and Firefox browsers as well as on BrowserStack with the apk WeVoteV2-4-5Feb28-731pm.apk for android devices as well as the iOS app from the AppStore.
